src/Jobs/RecurringAllGarbageCollectorJob.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace SilverStripe\GarbageCollector\Jobs;
+
+use Exception;
+use SilverStripe\Core\Config\Configurable;
+use SilverStripe\Core\Injector\Injectable;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\ORM\FieldType\DBDatetime;
+use SilverStripe\GarbageCollector\CollectorInterface;
+use SilverStripe\GarbageCollector\GarbageCollectorService;
+use Symbiote\QueuedJobs\Services\AbstractQueuedJob;
+use Symbiote\QueuedJobs\Services\QueuedJob;
+use Symbiote\QueuedJobs\DataObjects\QueuedJobDescriptor;
+use Symbiote\QueuedJobs\Services\QueuedJobService;
+
+/**
+ * @property CollectorInterface|null $collector
+ * @property int|null $batchSize
+ * @property array $remaining
+ * @property array $processors
+ * @property array $versions
+ * @property array $remainingVersions
+ */
+class RecurringAllGarbageCollectorJob extends AbstractQueuedJob
+{
+    use Configurable;
+    use Injectable;
+    /**
+     * Constructor
+     *
+     * @var string
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Defines the title of the job
+     * @return string
+     */
+    public function getTitle()
+    {
+        return sprintf("Garbage Collection processing for all collectors");
+    }
+
+    public function getJobType(): int
+    {
+        return QueuedJob::QUEUED;
+    }
+
+    public function setup(): void
+    {
+        $service = GarbageCollectorService::inst();
+        $this->totalSteps = count($service ->getCollectors());
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function process(): void
+    {
+        $logger = new \Monolog\Handler\TestHandler();
```

Comment:
Leftover from some tests perhaps? Or is there a reason to use test logger in live code?